src/main/scala/GithubRelease.scala

```diff
@@ -29,13 +22,13 @@ case object GithubRelease {
     lazy val ghreleaseNotes = settingKey[TagName => String]("Release notes for the given tag")
     lazy val ghreleaseTitle = settingKey[TagName => String]("The title of the release")
     lazy val ghreleaseIsPrerelease = settingKey[TagName => Boolean]("A function to determine release as a prerelease based on the tag name")
-    lazy val ghreleaseGithubCreds = settingKey[Option[GitHubCredentials]]("Credentials for accessing the GitHub API")
+    lazy val ghreleaseGithubToken = settingKey[Option[String]]("Credentials for accessing the GitHub API")
```

Comment:
Oh, shoot! I saw this after writing the same thing 😅 
Well, I'm glad to see that we are thinking in the same direction 👍